spec/support/features/session_helper.rb

```diff
@@ -79,7 +79,8 @@ def confirm_last_user
       User.last.update(
         confirmation_token: @raw_confirmation_token, confirmation_sent_at: Time.current
       )
-      visit "/users/confirmation?confirmation_token=#{@raw_confirmation_token}"
+      visit "/sign_up/create_password?confirmation_token=#{@raw_confirmation_token}"
+      # visit user_confirmation_url(confirmation_token: @raw_confirmation_token)
```

Comment:
this change seemed innocuous but broke some feature specs. Anyone know why? Commented it out so you can see the change I was trying to make